pkg/controllers/agent/yurtcluster_controller.go

```diff
@@ -156,6 +163,7 @@ func (r *YurtClusterReconciler) reconcileCloudNode(ctx context.Context, yurtClus
 
 func (r *YurtClusterReconciler) reconcileEdgeNode(ctx context.Context, yurtCluster *operatorv1alpha1.YurtCluster, node *corev1.Node) (ctrl.Result, error) {
 	if controllersutil.IsNodeConvertOrRevertCompleted(node, yurtCluster) {
+		r.recorder.Eventf(yurtCluster, corev1.EventTypeNormal, "ConvertCompleted", "edge node %s already convert complete.", node.Name)
```

Comment:
I'm worried that events like this will be created over and over again, as the controller is always listening for changes of the YurtCluster object.